processor/src/decorators/adv_stack_injectors.rs

```diff
@@ -338,6 +338,37 @@ where
 
         Ok(())
     }
+
+    /// Pushes values onto the advice stack which are required for verification of a DSA in Miden VM.
+    ///
+    /// Inputs:
+    ///   Operand stack: [PK, MSG, ...]
+    ///   Advice stack: [...]
+    ///   Advice map: {PK: [SK]}
+    ///
+    /// Outputs:
+    ///   Operand stack: [PK, MSG, ...]
+    ///   Advice stack: [DATA]
+    ///   Advice map: {PK: [SK]}
```

Comment:
I wonder if we should describe the transition as:
```
Inputs:
  Operand stack: [PK, MSG, ...]
  Advice stack: [...]

Outputs:
  Operand stack: [PK, MSG, ...]
  Advice stack: [SIG_DATA]
```
And then say that in the implementation of the stock advice provider (e.g., `MemAdviceProvider`), we also expect the advice map to contain the private key.